Documentation/ResultsStructure.md

```diff
@@ -0,0 +1,150 @@
+
+
+
+## R: results
+
+Results are ordered such that they represent 1 gait cycle, starting at the right side initial contact.
+If *S.misc.gaitmotion_type* is *FullGaitCycle*, the results use *S.solver.N_meshes* meshpoints.
+If *S.misc.gaitmotion_type* is *HalfGaitCycle*, the results use *2\*S.solver.N_meshes meshpoints*.
+
+### R.S: settings used to generate these results
```

Comment:
If we would still take all the settings out of the ReadMe, you can here refer to the file containing all settings.